content/01.main_content.md

```diff
@@ -13,4 +25,17 @@ Proxy predictions of ethnicity via name origin are more difficult still; the cho
   For instance, these classifiers are usually unable to distinguish if names of Hispanic origin come from the Iberian Peninsula or from Latin America [@doi:10.1016/j.cels.2021.07.007]. 
   If a target and background population had identical probabilistic proportions of Hispanic names, but the target group primarily consisted of individuals from Spain and Portugal, underrepresentation of Latin American scientists would then go unnoticed in an exclusively computational analysis.
 
-Considering these things, we propose the following recommendations for best practices in auditing scientific systems:
+Recognizing the aforementioned shortcomings, we propose the following recommendations for the creation and deployment of automated auditing tools:
+  1. *Transparency.* Publicly provide all tools, code, and data used in the analysis.
+  This is to enable public scrutiny and transparency to those being audited and those whose data you are using.
+  If the data is private, we recommend providing de-identified or aggregated data. 
+
+  2. *Individuals know best.* Self-identified demographic information should be used in preference to automated predictions.
+  Additionally, use the most granular and representative groups possible.
```

Comment:
I think this needs fleshing out. Are we recommending they have more descriptive groups for people to select from in the self-ID process? Or to use the most granular possible when training/running a model? We've already talked about the limits of granularity in these models.

Reply:
yes, good point. I reworded it a bit in: https://github.com/greenelab/disparities_opinion/pull/3/commits/14114d44aa176039d29d5d339c771f487ba0773c

I mostly moved the granularity part of point 2 to point 3.